zerver/webhooks/facebook/view.py

```diff
@@ -0,0 +1,123 @@
+from typing import Any, Dict, Optional, Text
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message, create_stream_if_needed
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success, json_error
+from zerver.models import UserProfile
+import json
+
+class UnknownObjectType(Exception):
+    pass
+
+def user_object(payload: Dict[Text, Any]) -> Text:
+    field = payload['entry']['changes']['field']
+    message = "Changed **{_field}**".format(_field=field)
+    if field == "email":
+        if payload['entry']['changes']['value'] is not None:
+            message = message + '\nTo: *{_email}*'.format(
+                _email=payload['entry']['changes']['value']
+            )
+    return message
+
+def page_object(payload: Dict[Text, Any]) -> Text:
+    field = payload['entry']['changes']['field']
+
+    message = ''
+    if field == 'conversations':
+        message = message + 'Updated **conversations**'
+        message = message + '\n[Open conversations...](https://www.facebook.com/' \
+                            '{_page_id}/{_thread_id})'.format(
+                                _page_id=payload['entry']['changes']['value']['page_id'],
+                                _thread_id=payload['entry']['changes']['value']['thread_id']
+                            )
+    elif field == 'website':
+        message = message + 'Changed **website**'
+    return message
+
+def permissions_object(payload: Dict[Text, Any]) -> Text:
+    field = payload['entry']['changes']['field']
+    message = '**{_field} permission** changed'.format(_field=field)
```

Comment:
I would suggest just using `field=field` instead of `_field`.

Reply:
I didn't know that can use same name.

I fixed it!